Jenkinsfile

```diff
@@ -385,6 +385,17 @@ lock(resource: "build-${params.STREAM}") {
             ]
         }
 
+	stage('Fork S390X Pipeline') {
+            build job: 'multi-arch-pipeline', wait: false, parameters: [
+                booleanParam(name: 'FORCE', value: params.FORCE),
+                booleanParam(name: 'MINIMAL', value: params.MINIMAL),
+                string(name: 'FCOS_CONFIG_COMMIT', value: fcos_config_commit),
+                string(name: 'STREAM', value: params.STREAM),
+                string(name: 'VERSION', value: newBuildID),
+                string(name: 'ARCH', value: 's390x')
+            ]
+        }
+
```

Comment:
This looks pretty similar to what is being done for `aarch64`, but is missing `COREOS_ASSEMBLER_IMAGE`.  Is there a reason it was omitted?

Reply:
I did this 2months ago as instructed by Dusty, Not sure if I missed the COREOS_ASSEMBER_IMAGE parameter. I'll add push again.

Reply:
@miabbott @dustymabe  I've made the changes as suggested.